### DISCUSSION TITLE
Small issue, not enough to be a bug. And an idea

### DISCUSSION THREAD

**Discussion by Gilbals** [massif-press/compcon#2288] [Archive]
I've been sitting on this system for a while and just now started reading it and getting into it. I especially love the work done here with COMP/CON. However there is a small visual issue that bothers me and probably only me.  In the Encounter Toolkit, the nav bar at the top for COMP/CON cuts off other text fields and obfuscates things albeit it super minimally. 

As for the idea, it'd be rather large. A blank slate mech builder as a GM tool, for custom building mechs on the fly for homebrew.

**Comment by msprijatelj** [comment]
Yeah the text field getting cut off I think stems from an old version of Vuetify that Netlify refuses to update. I believe this is the case because when I do a clean npm install, my local deployment looks fine. I’m hopeful the CompCon v3 update will take care of that issue. 

I could see a Blank Slate being useful for NPCs mostly? Being able to mix and match NPC parts like Legos, for example. Though that can be sorta done in existing CompCon, though it still needs a baseline NPC class chosen before customizing the hell out of it.

**Reply by Gilbals** [reply]
I didn't realize the NPC thing, but also yeah I'd like for mechs as well, for homebrew. Instead of learning how to make an entire LCP for a mech design that players could potentially get their hands on, just being able to build one in COMP/CON itself and share as simply as pilot data would be super handy.

**Reply by msprijatelj** [reply]
So you’re hoping for something like https://cc-lcp-editor.netlify.app/#/, but embedded in compcon?

**Reply by Gilbals** [reply]
Potentially, yeah. Though I'd just be happy with a blank option for Cores to make your own and the ability to edit base Frame stats. I did look into the LCP editor and it seems good, but early.

**Reply by jarena3** [reply]
I probably won't be pulling the lcp editor into comp/con, but after v3 is out my NPC data will be more or less fully stable so I'm going to be taking a in-depth pass at the editor and linking or otherwise exposing it in C/C, along with better campaign tools. Maybe I'll create an editor mode specifically for exotics that can ride along in campaign data?
